README.md

```diff
@@ -1,3 +1,62 @@
 # orchestrator-relayer
 
-Contains the implementation of the QGB orchestrator and relayer
+Contains the implementation of the QGB orchestrator and relayer.
+
+The orchestrator is the software that signs the QGB attestations, and the relayer is the one that relays them to the target EVM chain.
+
+For a high-level overview of how the QGB works, check [here](https://github.com/celestiaorg/quantum-gravity-bridge/tree/76efeca0be1a17d32ef633c0fdbd3c8f5e4cc53f#how-it-works) and [here](https://blog.celestia.org/celestiums/).
+
+## Install
+
+1. [Install Go](https://go.dev/doc/install) 1.19
+2. Clone this repo
+3. Install the QGB CLI
+
+ ```shell
+make install
+```
+
+## Usage
+
+```sh
+# Print help
+qgb --help
+```
+
+## How to run
+
+If you are a Celestia-app validator, all you need to do is run the orchestrator. Check [here](https://github.com/celestiaorg/orchestrator-relayer/blob/main/docs/orchestrator.md) for more details.
+
+If you want to run a relayer or target a new EVM chain, you will need to deploy a new QGB contract and run a relayer. Check [here](https://github.com/celestiaorg/orchestrator-relayer/blob/main/docs/relayer.md) for relayer docs and [here](https://github.com/celestiaorg/orchestrator-relayer/blob/main/docs/deploy.md) for how to deploy a new QGB contract.
```

Comment:
```suggestion
If you want to post commitments on an EVM chain, you will need to deploy a new QGB contract and run a relayer. Check [here](https://github.com/celestiaorg/orchestrator-relayer/blob/main/docs/relayer.md) for relayer docs and [here](https://github.com/celestiaorg/orchestrator-relayer/blob/main/docs/deploy.md) for how to deploy a new QGB contract.
```

Reply:
this felt a bit less repetitive and a bit more descriptive, but defifnitely leaving it as optional